tools/cli/admin_cluster_commands.go

```diff
@@ -125,17 +127,18 @@ func AdminRebalanceStart(c *cli.Context) {
 
 	resp, err := client.StartWorkflowExecution(tcCtx, request)
 	if err != nil {
-		ErrorAndExit("Failed to start failover workflow", err)
+		return ErrorAndPrint("Failed to start failover workflow", err)
 	}
 	fmt.Println("Rebalance workflow started")
 	fmt.Println("wid: " + workflowID)
 	fmt.Println("rid: " + resp.GetRunID())
+	return nil
 }
 
-func AdminRebalanceList(c *cli.Context) {
+func AdminRebalanceList(c *cli.Context) error {
 	c.Set(FlagWorkflowID, failovermanager.RebalanceWorkflowID)
-	c.GlobalSet(FlagDomain, common.SystemLocalDomainName)
-	ListWorkflow(c)
+	c.Set(FlagDomain, common.SystemLocalDomainName)
```

Comment:
these also return errors / errcheck catches it

Reply:
fixed